x-pack/plugins/snapshot_restore/server/routes/api/policy.ts

```diff
@@ -203,7 +203,7 @@ export function registerPolicyRoutes({
       try {
         const response = await clusterClient.asCurrentUser.indices.resolveIndex({
           name: '*',
-          expand_wildcards: 'all',
+          expand_wildcards: 'open',
```

Comment:
We now want to ignore hidden indices in the create policy flow, so `open` will give us all non-hidden ds and indices. See [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-resolve-index-api.html#resolve-index-api-query-params)